apps/backend-e2e/src/e2e/auth.cy.ts

```diff
@@ -15,140 +15,87 @@ describe('AUTHENTIFICATION TESTS', () => {
       - Check self-profile of editor user with wrong jwt.
       - Check self-profile with correct jwt.
    */
-
-  /*
-      SENSIBLE DATA MUST BE SENT WITH COMMAND LINE AND OVERRIDE FOLLOWING DEFAULT ONES :
-   */
-  let jwtUser = {};
-  let idUserAdmin;
-  let idUserEditor;
-
-  /*
-      TESTS
-   */
-  it('Auth -> Connecting with incorrect user.', () => {
+  it('Should fail when trying to connect without any credentials 1/2.', () => {
     cy.request({
       method: 'POST',
       url: '/api/auth/login',
-      body: {
-        email: faker.internet.email(),
-        password: faker.internet.password(),
-      },
+      body: {},
       failOnStatusCode: false,
     }).then((response) => {
       expect(response.status).to.eq(401);
     });
   });
-  it('Auth -> Connecting with proper user admin but wrong admin password.', () => {
+  it('Should fail when trying to connect without any credentials 2/2.', () => {
     cy.request({
-      method: 'POST',
+      method: 'GET',
       url: '/api/auth/login',
-      body: {
-        email: 'admin@example.org',
-        password: faker.internet.password(),
-      },
+      body: {},
       failOnStatusCode: false,
     }).then((response) => {
-      expect(response.status).to.eq(401);
+      expect(response.status).to.eq(404);
     });
   });
-  it('Auth -> Connecting correctly with admin user.', () => {
+  it('Should fail when trying to connect without any data 1/2.', () => {
     cy.request({
       method: 'POST',
       url: '/api/auth/login',
-      body: {
-        username: 'admin@example.org',
-        password: 'admin',
-      },
       failOnStatusCode: false,
     }).then((response) => {
-      jwtUser = response.body.access_token;
-      idUserAdmin = response.body.user_id;
-      expect(response.status).to.eq(201);
+      expect(response.status).to.eq(401);
     });
   });
-  it('Auth -> Check self-profile of admin user with wrong jwt.', () => {
+  it('Should fail when trying to connect without any data 2/2.', () => {
     cy.request({
       method: 'GET',
-      url: `/api/users/${idUserAdmin}`,
+      url: '/api/auth/login',
       failOnStatusCode: false,
-      auth: {
-        bearer: 'incorrect_jwt',
-      },
     }).then((response) => {
-      expect(response.status).to.eq(403);
+      expect(response.status).to.eq(404);
     });
   });
-  it('Auth -> Check self-profile of admin user with correct jwt.', () => {
+  it('Connecting with incorrect user.', () => {
```

Comment:
```suggestion
  it('Should fail to connect with incorrect user.', () => {
```

Sentence more descriptive : 
- follows test naming convention `it should ...`
- explicitly indicates the result

Reply:
As I said earlier this code was juste moved over (and disclaimer -- I'm not the author :sweat_smile: )
But you're right about the naming convention : it's the idiomatic way of chai, etc.
However If I had to refactor this, I would instead describe testing scenario rather than being too close to implementation details and __even if__ it breaks the `it` convention. 
This convention is inherited from __unit testing__ library and is too restrictive to describe something closer to user story.